Move npm-check commands from style-guide.md to pr-checks.md (#7191)
open-telemetry/opentelemetry.io: content/en/docs/contributing/style-guide.md

[`.cspell.yml`](https://github.com/open-telemetry/opentelemetry.io/blob/main/.cspell.yml)
file.

> If `cspell` indicates an `Unknown word` error, verify if you wrote that word
> correctly. If so, add this word to the `cSpell:ignore` section at the top of
> your file. If no such section exists, you can add it to the front matter of a
> Markdown file:

Reviewer: ```suggestion
If `cspell` indicates an "Unknown word" error, check whether you wrote the word
correctly. If so, add the word to the `cSpell:ignore` section at the top of your
file. If no such section exists, you can add it to the front matter of a
Markdown file:
```
